interfaces/builtin/opengl.go

```diff
@@ -108,6 +108,7 @@ const openglConnectedPlugAppArmor = `
 @{PROC}/modules r,
 /dev/nvidia* rw,
 unix (send, receive) type=dgram peer=(addr="@nvidia[0-9a-f]*"),
+unix (send, receive) type=stream peer=(label="Xorg"),
```

Comment:
Do we need this?
```suggestion
unix (send, receive) type=stream peer=(label="{Xorg,unconfined}"),
```

Reply:
hm no w8, this is wrong after all, if anything this should be added to the x11 interface

Reply:
@aleasto can you clarify with JJ if the case for unconfined is needed. I'm not comfortable mergig it without this given that we don't have automated tests on nvidia hardware anywhere.

Reply:
The way I understand this, this is fine as-is and is also future proof until delegation is implemented enough that unconfined can be confined by another profile.